satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -1184,10 +1274,11 @@ datasets:
 
   pressure:
     name: pressure
+    long_name: Wind Pressure
+    standard_name: wind_pressure
```

Comment:
I would suggest changing to "air_pressure"

Reply:
done